src/TodoItemCreater.jsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+
+const TodoItemCreater = ({ onSubmit }) => {
+  const [inputValue, setInputValue] = useState('');
+
+  const handleChangeInput = (e) => setInputValue(e.target.value);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    if (!inputValue) {
+      // eslint-disable-next-line no-alert
+      alert('할 일을 입력해 주세요!');
+      return;
+    }
+
+    if (onSubmit) {
+      onSubmit(inputValue);
+    }
+    setInputValue('');
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <input value={inputValue} placeholder="할 일을 입력해 주세요" onChange={handleChangeInput} />
```

Comment:
```suggestion
      <input
        value={inputValue}
        placeholder="할 일을 입력해 주세요"
        onChange={handleChangeInput} 
      />
```

표현해야할 정보가 많아질 때 세로로 정렬하면 가독성이 좋아져요